### ISSUE TITLE
1.19 & newer Clients can't click/accept Quests in chat

### ISSUE THREAD

**Issue by TheLaoming** [PikaMug/Quests#2032]
### What is your Quests version / build number (do *not* say "latest")?

Tested on two servers:

**Quests version:** 4.5.1-b348 (but tested with all versions above up to 4.6.0).
**Server version:** 1.18.2 Paper-388
&
**Quests version:** 4.6.0-b361
**Server version:** 1.19.2 Paper-280

### Describe the problem and how to replicate it.

**About my issue:** 
So, there's a bug related #2017 but it's not the same issue, it's just similar. All 1.19+ clients aren't able to click on Quests in chat. For example when you click on NPC, you get a list of 7 quests, when you're normally supossed to be able to click them. But it doesn't work.  You can't click on them if you're on **1.19+**. Clients before 1.19 are able to click on quests (1.18, 1.17 etc.). This issue is only for quests list given from NPC, since confirm message like in #2017  seems to be fixed and works fine for newer versions.
So I can click on Yes / No, but not on Quests itself. That's the issue.

Basically, I run my server on version 1.18 and block everyone who want to join from 1.19, due to this issue...Because of a single problem ruining the enjoyment of the game. It's definitely something with chat reports or new system of handling commands.
Hope it will be possible to fix it.

### Any console errors to share?

```shell
Nothing at all
```

**Comment by PikaMug** [comment]
Could you check whether giving users the `quests.choice` permission resolves this issue?

**Comment by TheLaoming** [comment]
> Could you check whether giving users the `quests.choice` permission resolves this issue on Quests 4.6.0?
> 
> As for 4.5.1, that version is not compatible with the way 1.19 handles clickable chat. Upgrade to 4.6.0.

Tested with OP permission or this specific permission without op, doesn't work. I'm able to click only on Yes / No, not on quests.
Image: [https://i.imgur.com/HLpeoUS.png](url)
Also this time tested on 1.19.2 version and with 4.6.0 build from spigot.
